system-contracts/hardhat.config.ts

```diff
@@ -6,8 +6,8 @@ import "@nomiclabs/hardhat-ethers";
 import "hardhat-typechain";
 
 // This version of system contracts requires a pre release of the compiler
-const COMPILER_VERSION = "1.5.0";
-const PRE_RELEASE_VERSION = "prerelease-a167aa3-code4rena";
+const COMPILER_VERSION = "fix-output-directory"; // 1.5.5
+const PRE_RELEASE_VERSION = "72ea6d2";
 function getZksolcUrl(): string {
```

Comment:
this `getZksolcUrl` thing was needed before when the prerelease version had not yet been published, now we can remove it, could you please switch to just providing version number in hardhat config ts? similarly to how it is done in l2-contracts

Reply:
@StanislavBreadless we are using the pre-release version in this PR

Reply:
This prerelease fixes a bug with `--output-dir`